charts/pvault-server/values.yaml

```diff
@@ -41,6 +41,10 @@ replicaCount: 1
 updateStrategy:
   type: RollingUpdate
 
+## @param minReadySeconds Minimum number of seconds for the pod to be considered available.
+##
+minReadySeconds: 10
```

Comment:
@nirhaas thoughts on 10s as default ?

Reply:
I think it's ok. Can't think of a compelling reason to change to a higher or lower value